CONTRIBUTING.md

```diff
@@ -84,7 +84,7 @@ We use GitHub issues to track bugs exclusively. We have documented our issue han
 
 ### Merging upstream master into your fork master
 
-From time to time, your fork will get out of sync with the upstream remote. Use the following commands to get the master branch of your fork up up to date.
+We use GitHub issues to track bugs exclusively. We have documented our issue handling processes in the [Issues wiki](https://github.com/facebook/react-native/wiki/Triaging-GitHub-Issues).
```

Comment:
We're getting rid of our advice on how to sync with FB's upstream repo and replacing it with the upstream issue wiki? I'd imagine we at least want to link the rn-macos Issues template if we change this right?

Reply:
This added line is just a copy of another line that already appears in this file. I'm assuming this was an accident? Either way, we should definitely revert it back.